client/src/modules/patients/groups/templates/action.cell.html

```diff
@@ -0,0 +1,35 @@
+<div
+  class="ui-grid-cell-contents text-action"
+  data-row="{{row.entity.name}}"
+  uib-dropdown dropdown-append-to-body>
+
+  <a uib-dropdown-toggle href data-action="open-dropdown-menu">
+    <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+    <span class="caret"></span>
+  </a>
+
+  <ul data-row-menu="{{ row.entity.name }}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
+    <li>
+      <a data-method="edit-record" ng-click="grid.appScope.createUpdateGroupModal(row.entity)" href>
+        <i class="fa fa-edit"></i> <span translate>FORM.BUTTONS.EDIT</span>
+      </a>
+    </li>
+    <li>
+      <a class="btn btn-xs btn-default" ui-sref="patientRegistry({
```

Comment:
Could you remove this `class="btn btn-xs btn-default"` it is not necessary for a drop down list.